File: app/src/main/java/com/fdb/mvpdemo/widget/holder/HouseCollectHolder.java
package com.fdb.mvpdemo.widget.holder;

import android.view.View;
import android.widget.TextView;

import com.fdb.mvpdemo.R;
import com.fdb.mvpdemo.bean.HouseBean;
import com.fdb.mvpdemo.widget.base.BaseHolder;

import butterknife.BindView;

/**
 * Desc
 * Author dontlo
 * Date   2020/4/17.
 */
public class HouseCollectHolder extends BaseHolder<HouseBean> {
    @BindView(R.id.tv_name)
    TextView mTvName;

    public HouseCollectHolder(View itemView) {
        super(itemView);
    }

    @Override
    protected int getLayoutId() {
        return R.layout.item_house_collect;
    }

    @Override
    protected void onSetData() {
        mTvName.setText(mBean.communityName);
    }
}

File: app/src/main/java/com/fdb/mvpdemo/widget/holder/HouseCollectHolder.kt
package com.fdb.mvpdemo.widget.holder

import android.view.View
import com.fdb.mvpdemo.R
import com.fdb.mvpdemo.bean.HouseBean
import com.fdb.mvpdemo.widget.base.BaseHolder
import kotlinx.android.synthetic.main.item_house_collect.view.*

/**
 * Desc
 * Author dontlo
 * Date   2020/4/17.
 */
class HouseCollectHolder(itemView: View?) : BaseHolder<HouseBean?>(itemView){
    override fun getLayoutId(): Int {
        return R.layout.item_house_collect
    }

    override fun onSetData() {
        rootView.tv_name!!.text = mBean!!.communityName
    }
}